src/lib/events.js

```diff
@@ -44,7 +44,6 @@ export const ERROR_CODE = {
     FLASH_NOT_ENABLED: 'error_flash_not_enabled'
 };
 
-export const PREVIEW_LOAD_EVENT = '';
```

Comment:
Why are we removing this?

Reply:
It's not used anywhere AND it's an empty string

Reply:
We now use the below `LOAD_METRIC.previewLoadEvent`